### ISSUE TITLE
rdkit 2022.03.5 not able to be imported (MacOS, osx-arm64, conda-forge)

### ISSUE THREAD

**Issue by jrsacher** [rdkit/rdkit#5626]
**Describe the bug**
I am unable to import rdkit with version `2022.03.5` installed. If I downgrade to `2022.03.4`, it works as expected

**To Reproduce**
```
(base) ➜  ~ conda list | grep rdkit
rdkit                     2022.03.5        py39h042da20_0    conda-forge

(base) ➜  ~ python
Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:33) 
[Clang 13.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import rdkit
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'rdkit'
>>> 

(base) ➜  ~ conda install rdkit=2022.03.4 --force-reinstall
[output deleted for brevity]

(base) ➜  ~ conda list | grep rdkit
rdkit                     2022.03.4        py39he8580ed_0    conda-forge

(base) ➜  ~ python
Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:33) 
[Clang 13.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import rdkit
>>> rdkit.rdBase.rdkitVersion
'2022.03.4'
>>> 
```

**Expected behavior**
rdkit able to be imported

**Configuration (please complete the following information):**
 - RDKit version: `2022.03.5`
 - OS: macOS 12.6, osx-arm64
 - Python version (if relevant): `3.9.13`
 - Are you using conda? Yes, version `22.9.0`
 - If you are using conda, which channel did you install the rdkit from? `conda-forge`
 - If you are not using conda: how did you install the RDKit? n/a

**Comment by jrsacher** [comment]
Posted to the wrong repo, I'm assuming. Sorry! I'll close this here and reopen it in `conda-rdkit`.

**Comment by jrsacher** [comment]
Okay, it looks like `conda-rdkit` hasn't been touched in a while, so I'm reopening here. Let me know if it should go somewhere else.

**Comment by greglandrum** [comment]
I can't reproduce this on the mac that I use for testing, so it's not a general problem.
I'd try reinstalling 2022.03.5 in that environment... if that doesn't work then please try creating a new conda environment to see if it works for you there; if so then it's a sign that something has gotten borked in conda environment you're using.

**Comment by jrsacher** [comment]
Out of curiosity, I created a new conda environment with only rdkit. It installed python 3.10, and everything worked fine.

I created a brand new python 3.9 environment using `conda create -c conda-forge -n rdkit_39 python=3.9 rdkit`. After activating the environment, I get the same `ModuleNotFound` error.

I updated my existing environment from python 3.9 to 3.10, which switched the rdkit build from `2022.03.5-py39h042da20_0` to `2022.03.5-py310h936bf1d_0`. That seems to have fixed the issue.
